[CI] EsqlActionBreakerIT testTopNPushedToLucene failing

**Build Scans:**
- [elasticsearch-periodic #9108 / openjdk24_checkpart3_java-matrix](https://gradle-enterprise.elastic.co/s/hqeom3bblarqw)
- [elasticsearch-periodic-platform-support #9072 / windows-2022_checkpart3_platform-support-windows](https://gradle-enterprise.elastic.co/s/cjkace4k5wkia)
- [elasticsearch-periodic #9015 / encryption-at-rest](https://gradle-enterprise.elastic.co/s/6rfq4myzuckzy)
- [elasticsearch-periodic-platform-support #9023 / ubuntu-2404-aarch64_checkpart3_platform-support-arm](https://gradle-enterprise.elastic.co/s/s346vyynx2avy)
- [elasticsearch-pull-request #78091 / part-3](https://gradle-enterprise.elastic.co/s/e2qanraltqhhk)
- [elasticsearch-pull-request #78050 / part-3](https://gradle-enterprise.elastic.co/s/mjr2wo7xncdr2)
- [elasticsearch-periodic-platform-support #9001 / oraclelinux-9_platform-support-unix](https://gradle-enterprise.elastic.co/s/q5mptptpvc3rk)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:internalClusterTest" --tests "org.elasticsearch.xpack.esql.action.EsqlActionBreakerIT.testProjectWhere" -Dtests.seed=8D7BD72191F69394 -Dtests.locale=ce -Dtests.timezone=Etc/Universal -Druntime.java=24
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.action.EsqlActionBreakerIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(testTopNPushedToLucene),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.Exception: Test abandoned because suite timeout was reached.
```

**Issue Reasons:**
- [main] 7 failures in test testTopNPushedToLucene (0.8% fail rate in 866 executions)
- [main] 2 failures in step part-3 (0.6% fail rate in 317 executions)
- [main] 2 failures in pipeline elasticsearch-periodic (10.5% fail rate in 19 executions)
- [main] 2 failures in pipeline elasticsearch-pull-request (0.6% fail rate in 324 executions)
- [main] 3 failures in pipeline elasticsearch-periodic-platform-support (15.8% fail rate in 19 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

I think there is some double releases: 

```
  2> REPRODUCE WITH: ./gradlew ":x-pack:plugin:esql:internalClusterTest" --tests "org.elasticsearch.xpack.esql.action.EsqlActionBreakerIT" -Dtests.seed=8D7BD72191F69394 -Dtests.locale=en -Dtests.timezone=Etc/UTC -Druntime.java=24
  2> java.lang.AssertionError: Used bytes: [-216] must be >= 0
        at __randomizedtesting.SeedInfo.seed([8D7BD72191F69394]:0)
        at org.elasticsearch.common.breaker.ChildMemoryCircuitBreaker.addWithoutBreaking(ChildMemoryCircuitBreaker.java:207)
        at org.elasticsearch.common.util.BigArrays.adjustBreaker(BigArrays.java:539)
        at org.elasticsearch.common.util.AbstractArray.close(AbstractArray.java:29)
```

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -382,12 +382,6 @@ tests:
 - class: org.elasticsearch.index.codec.vectors.cluster.HierarchicalKMeansTests
   method: testHKmeans
   issue: https://github.com/elastic/elasticsearch/issues/130497
-- class: org.elasticsearch.xpack.esql.action.EsqlActionBreakerIT
-  method: testProjectWhere
-  issue: https://github.com/elastic/elasticsearch/issues/130504
-- class: org.elasticsearch.xpack.esql.action.EsqlActionBreakerIT
-  method: testTopNPushedToLucene
-  issue: https://github.com/elastic/elasticsearch/issues/130505
 - class: org.elasticsearch.gradle.LoggedExecFuncTest
   method: failed tasks output logged to console when spooling true
   issue: https://github.com/elastic/elasticsearch/issues/119509
@@ -406,12 +400,6 @@ tests:
 - class: org.elasticsearch.search.SearchWithRejectionsIT
   method: testOpenContextsAfterRejections
   issue: https://github.com/elastic/elasticsearch/issues/130821
-- class: org.elasticsearch.xpack.esql.action.EsqlActionBreakerIT
-  method: testExtractFields
-  issue: https://github.com/elastic/elasticsearch/issues/130501
-- class: org.elasticsearch.xpack.esql.action.EsqlActionBreakerIT
-  method: testRowStatsProjectGroupByInt
-  issue: https://github.com/elastic/elasticsearch/issues/131024
 - class: org.elasticsearch.xpack.esql.qa.multi_node.EsqlSpecIT
   method: test {lookup-join.MvJoinKeyOnFromAfterStats ASYNC}
   issue: https://github.com/elastic/elasticsearch/issues/131148

